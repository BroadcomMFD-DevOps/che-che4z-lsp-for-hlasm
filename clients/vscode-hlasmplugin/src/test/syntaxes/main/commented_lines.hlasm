* dadsa                                                                - 
               asd as d
LABEL    INSTR   OP THIS LINE SHOULD NOT BE AFFECTED
.* REMARK                                                              -
               dafaf
LABEL    INSTR   OP THIS LINE SHOULD NOT BE AFFECTED
.* REMARK                                                              -
               dafaf                                                   -

.* REMARK                                                              -
               dafaf                                                   -

LABEL    INSTR   OP THIS LINE SHOULD NOT BE AFFECTED

*        Multiline remarks                                             -        
               remarks                                                 -12345678
               remarks                                                 -        
               remarks                                                 -        
               remarks                                                 -        
               remarks                                                 -        
               remarks                                                          
LABEL    INSTR   OP THIS LINE SHOULD NOT BE AFFECTED
