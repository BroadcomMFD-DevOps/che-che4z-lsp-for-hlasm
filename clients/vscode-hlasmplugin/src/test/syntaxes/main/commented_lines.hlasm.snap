>* dadsa                                                                - 
#^ source.hlasm comment.hlasm
# ^ source.hlasm
#  ^^^^^ source.hlasm comment.hlasm
#       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^ source.hlasm invalid.ignored
>               asd as d
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^ source.hlasm comment.hlasm
#                  ^ source.hlasm
#                   ^^ source.hlasm comment.hlasm
#                     ^ source.hlasm
#                      ^ source.hlasm comment.hlasm
>LABEL    INSTR   OP THIS LINE SHOULD NOT BE AFFECTED
#^^^^^ source.hlasm support.class.hlasm
#     ^^^^ source.hlasm
#         ^^^^^ source.hlasm entity.name.function.hlasm
#              ^^^ source.hlasm
#                 ^^ source.hlasm punctuation.definition.tag.hlasm
#                   ^ source.hlasm
#                    ^^^^ source.hlasm comment.hlasm
#                        ^ source.hlasm
#                         ^^^^ source.hlasm comment.hlasm
#                             ^ source.hlasm
#                              ^^^^^^ source.hlasm comment.hlasm
#                                    ^ source.hlasm
#                                     ^^^ source.hlasm comment.hlasm
#                                        ^ source.hlasm
#                                         ^^ source.hlasm comment.hlasm
#                                           ^ source.hlasm
#                                            ^^^^^^^^ source.hlasm comment.hlasm
>.* REMARK                                                              -
#^^ source.hlasm comment.hlasm
#  ^ source.hlasm
#   ^^^^^^ source.hlasm comment.hlasm
#         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
>               dafaf
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^^^ source.hlasm comment.hlasm
>LABEL    INSTR   OP THIS LINE SHOULD NOT BE AFFECTED
#^^^^^ source.hlasm support.class.hlasm
#     ^^^^ source.hlasm
#         ^^^^^ source.hlasm entity.name.function.hlasm
#              ^^^ source.hlasm
#                 ^^ source.hlasm punctuation.definition.tag.hlasm
#                   ^ source.hlasm
#                    ^^^^ source.hlasm comment.hlasm
#                        ^ source.hlasm
#                         ^^^^ source.hlasm comment.hlasm
#                             ^ source.hlasm
#                              ^^^^^^ source.hlasm comment.hlasm
#                                    ^ source.hlasm
#                                     ^^^ source.hlasm comment.hlasm
#                                        ^ source.hlasm
#                                         ^^ source.hlasm comment.hlasm
#                                           ^ source.hlasm
#                                            ^^^^^^^^ source.hlasm comment.hlasm
>.* REMARK                                                              -
#^^ source.hlasm comment.hlasm
#  ^ source.hlasm
#   ^^^^^^ source.hlasm comment.hlasm
#         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
>               dafaf                                                   -
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^^^ source.hlasm comment.hlasm
#                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
>
>.* REMARK                                                              -
#^^ source.hlasm comment.hlasm
#  ^ source.hlasm
#   ^^^^^^ source.hlasm comment.hlasm
#         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
>               dafaf                                                   -
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^^^ source.hlasm comment.hlasm
#                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
>
>LABEL    INSTR   OP THIS LINE SHOULD NOT BE AFFECTED
#^^^^^ source.hlasm support.class.hlasm
#     ^^^^ source.hlasm
#         ^^^^^ source.hlasm entity.name.function.hlasm
#              ^^^ source.hlasm
#                 ^^ source.hlasm punctuation.definition.tag.hlasm
#                   ^ source.hlasm
#                    ^^^^ source.hlasm comment.hlasm
#                        ^ source.hlasm
#                         ^^^^ source.hlasm comment.hlasm
#                             ^ source.hlasm
#                              ^^^^^^ source.hlasm comment.hlasm
#                                    ^ source.hlasm
#                                     ^^^ source.hlasm comment.hlasm
#                                        ^ source.hlasm
#                                         ^^ source.hlasm comment.hlasm
#                                           ^ source.hlasm
#                                            ^^^^^^^^ source.hlasm comment.hlasm
>
>*        Multiline remarks                                             -        
#^ source.hlasm comment.hlasm
# ^^^^^^^^ source.hlasm
#         ^^^^^^^^^ source.hlasm comment.hlasm
#                  ^ source.hlasm
#                   ^^^^^^^ source.hlasm comment.hlasm
#                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               remarks                                                 -12345678
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^^^^^ source.hlasm comment.hlasm
#                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               remarks                                                 -        
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^^^^^ source.hlasm comment.hlasm
#                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               remarks                                                 -        
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^^^^^ source.hlasm comment.hlasm
#                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               remarks                                                 -        
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^^^^^ source.hlasm comment.hlasm
#                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               remarks                                                 -        
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^^^^^ source.hlasm comment.hlasm
#                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               remarks                                                          
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^^^^^ source.hlasm comment.hlasm
#                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>LABEL    INSTR   OP THIS LINE SHOULD NOT BE AFFECTED
#^^^^^ source.hlasm support.class.hlasm
#     ^^^^ source.hlasm
#         ^^^^^ source.hlasm entity.name.function.hlasm
#              ^^^ source.hlasm
#                 ^^ source.hlasm punctuation.definition.tag.hlasm
#                   ^ source.hlasm
#                    ^^^^ source.hlasm comment.hlasm
#                        ^ source.hlasm
#                         ^^^^ source.hlasm comment.hlasm
#                             ^ source.hlasm
#                              ^^^^^^ source.hlasm comment.hlasm
#                                    ^ source.hlasm
#                                     ^^^ source.hlasm comment.hlasm
#                                        ^ source.hlasm
#                                         ^^ source.hlasm comment.hlasm
#                                           ^ source.hlasm
#                                            ^^^^^^^^ source.hlasm comment.hlasm
>